vectordraw.js

```diff
@@ -30,16 +31,25 @@ var VectorDraw = function(element_id, settings) {
 VectorDraw.prototype.template = _.template([
     '<div class="jxgboard" style="width:<%= width %>px; height:<%= height %>px;" />',
     '<div class="menu">',
-    '    <div>',
+    '    <div class="controls">',
     '        <select>',
     '        <% vectors.forEach(function(vec, idx) { %>',
     '            <option value="<%= idx %>"><%= vec.description %></option>',
     '        <% }) %>',
     '        </select>',
     '        <button class="add-vector">Add Selected Force</button>',
-    '        <button class="reset">Clear All</button>',
-    '        <button class="undo">Undo</button>',
-    '        <button class="redo">Redo</button>',
+    '        <button class="reset">Reset</button>',
+    '        <button class="undo" title="Undo"><span class="fa fa-undo" /></button>',
+    '        <button class="redo" title="redo"><span class="fa fa-repeat" /></button>',
+    '    </div>',
+    '    <div class="vector-properties">',
```

Comment:
@mtyaka this block is never updated - it always shows 3.4 and 74 degrees. Probably there are some code missing, or it should be removed.